source/string_tools.f90

```diff
@@ -293,15 +293,16 @@ function chr_expandBrackets(theString) result(theResult)
   theBuffer = " "//theString//" "
   bPos(:,:) = 0
   lSP  = 0
+  tSP  = 0
```

Comment:
forgot `! -Wmaybe-uninitialized`

Reply:
I added those flags in places where I zeroed values without being 100% sure it was correct. This is my code, so I know what it's supposed to do.